webapp/src/accessibleblocks.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import * as data from "./data";
+import * as sui from "./sui";
+
+export interface AccessibleBlocksInfoState {
+    shown?: boolean;
+}
+
+export class AccessibleBlocksInfo extends data.Component<{}, AccessibleBlocksInfoState> {
+    constructor(props: {}) {
+        super(props);
+        this.state = { shown: false }
+        this.handleOnClose = this.handleOnClose.bind(this);
+    }
+
+    handleOnClose() {
+        this.setState({ shown: true })
+    }
+
+    render() {
+        const shown = this.state.shown;
+        if (shown) return <div />;
+        return <sui.Modal isOpen={!this.state.shown} onClose={this.handleOnClose} closeIcon={true}
+            dimmer={true} header={lf("Move, edit, and insert blocks using your keyboard")}>
+            <div className="ui equal width grid">
+                <div className="column">
+                    <h3>{lf("Navigate")}</h3>
+                    <span className="ui small">{lf("Use the WASD keys to move between blocks. Hold shift to move on the workspace.")}</span>
```

Comment:
What is the story for non-US keyboards?

Reply:
Could we make WASD a localized string and pick keys from there?

Reply:
not handled for now--i think the hope is if they integrate screen reader support we can use the arrow keys for everything

i think the most flexible solution is probably to expose some kind of UI to let users remap the controls based on whatever keyboard they have plugged in, but i'm also hoping we can just switch to arrow keys haha

Reply:
This is experimentation feature, lets postpone this problem for now. It unrelated to this checkin